### PR TITLE
Refactor/classification output

### PR DESCRIPTION
**Pull request by dilyararimovna**
change keras_classification_model output to probabilities, change metrics to more standard view

### CODE REVIEW

File: deeppavlov/models/go_bot/bot.py [hunk 1]
```diff
@@ -117,8 +117,8 @@ def __init__(self,
 
         self.intents = []
         if callable(self.intent_classifier):
-            # intent_classifier returns (y_labels, y_probs)
-            self.intents = list(self.intent_classifier(["hi"])[1][0].keys())
+            # intent_classifier returns y_probas
+            self.intents = self.intent_classifier.classes
```
Comment (yoptar) [inline deeppavlov/models/go_bot/bot.py]:
```suggestion
            self.intents = self.intent_classifier.get_main_component().classes
```

Comment (dilyararimovna) [reply deeppavlov/models/go_bot/bot.py]: Thank you! fixed

File: deeppavlov/models/classifiers/keras_classification_model.py [hunk 2]
```diff
@@ -56,50 +53,51 @@ class KerasClassificationModel(KerasModel):
...
+                         optimizer=optimizer,
+                         loss=loss,
+                         lear_rate=lear_rate,
+                         lear_rate_decay=lear_rate_decay,
+                         last_layer_activation=last_layer_activation,
+                         restore_lr=restore_lr,
+                         classes=classes,
+                         **kwargs)
+
+        if not(classes is None):
```
Comment (yoptar) [inline deeppavlov/models/classifiers/keras_classification_model.py]:
```suggestion
        if classes is not None:
```

Comment (dilyararimovna) [reply deeppavlov/models/classifiers/keras_classification_model.py]: fixed

**Pull request merged**